user-interface/src/data-verification/ConsolidationOrderModal.test.tsx

```diff
@@ -101,17 +101,29 @@ describe('ConsolidationOrderModalComponent', () => {
     // Select consolidation type
     const radioAdministrative = screen.queryByTestId(`radio-administrative-${id}`);
     const radioSubstantive = screen.queryByTestId(`radio-substantive-${id}`);
+    const radioAdministrativeClickTarget = screen.queryByTestId(
+      `radio-administrative-${id}-click-target`,
+    );
+    const radioSubstantiveClickTarget = screen.queryByTestId(
+      `radio-substantive-${id}-click-target`,
+    );
 
     expect(radioAdministrative).toBeInTheDocument();
     expect(radioSubstantive).toBeInTheDocument();
 
-    fireEvent.click(radioAdministrative!);
-    expect(radioAdministrative).toBeChecked();
-    expect(radioSubstantive).not.toBeChecked();
+    fireEvent.click(radioAdministrativeClickTarget!);
+
+    await waitFor(() => {
+      expect(radioAdministrative).toBeChecked();
+      expect(radioSubstantive).not.toBeChecked();
+    });
+
+    fireEvent.click(radioSubstantiveClickTarget!);
```

Comment:
I would suggest picking between 114 and 121 and removing the rest of these lines. Note that the expect is currently written in favor of line 121.